src/dialogs/more-info/controls/more-info-update.ts

```diff
@@ -0,0 +1,193 @@
+          )}
+        </div>
+        <div class="value">${this.stateObj.attributes.current_version}</div>
+      </div>
+      <div class="row">
+        <div class="key">
+          ${this.hass.localize(
+            "ui.dialogs.more_info_control.update.latest_version"
+          )}
+        </div>
+        <div class="value">${this.stateObj.attributes.latest_version}</div>
+      </div>
+
+      ${this.stateObj.attributes.release_url
+        ? html`<div class="row">
+            <div class="key">
+              <a
+                href=${this.stateObj.attributes.release_url}
+                target="_blank"
+                rel="noreferrer"
+              >
+                ${this.hass.localize(
+                  "ui.dialogs.more_info_control.update.release_announcement"
+                )}
+              </a>
+            </div>
+          </div>`
+        : ""}
+      ${this.stateObj.attributes.release_summary
+        ? html`<hr />
+            <ha-markdown
+              .content=${this.stateObj.attributes.release_summary}
+            ></ha-markdown> `
+        : ""}
+      ${supportsFeature(this.stateObj, UPDATE_SUPPORT_BACKUP)
+        ? html`<hr />
+            <ha-formfield
+              .label=${this.hass.localize(
+                "ui.dialogs.more_info_control.update.create_backup"
+              )}
+            >
+              <ha-checkbox
+                checked
+                .disabled=${updateIsInstalling(this.stateObj)}
+              ></ha-checkbox>
+            </ha-formfield> `
+        : ""}
+      <hr />
+      <div class="actions">
+        <mwc-button
+          @click=${this._handleSkip}
+          .disabled=${skippedVersion ||
+          this.stateObj.state === "off" ||
+          updateIsInstalling(this.stateObj)}
+        >
+          ${this.hass.localize("ui.dialogs.more_info_control.update.skip")}
+        </mwc-button>
+        ${supportsFeature(this.stateObj, UPDATE_SUPPORT_INSTALL)
+          ? html`
+              <mwc-button
+                @click=${this._handleInstall}
+                .disabled=${(this.stateObj.state === "off" &&
+                  !skippedVersion) ||
+                updateIsInstalling(this.stateObj)}
+              >
+                ${this.hass.localize(
+                  "ui.dialogs.more_info_control.update.install"
+                )}
+              </mwc-button>
+            `
+          : ""}
+      </div>
+    `;
+  }
+
+  get _shouldCreateBackup(): boolean | null {
+    if (!supportsFeature(this.stateObj!, UPDATE_SUPPORT_BACKUP)) {
+      return null;
+    }
+    const checkbox = this.shadowRoot?.querySelector("ha-checkbox");
+    if (checkbox) {
+      return checkbox.checked;
+    }
+    return true;
+  }
+
+  private _handleInstall(): void {
+    this.hass.callService("update", "install", {
+      entity_id: this.stateObj!.entity_id,
+      backup: this._shouldCreateBackup,
+      version: supportsFeature(this.stateObj!, UPDATE_SUPPORT_SPECIFIC_VERSION)
```

Comment:
If an update entity supports specific version, can we not pass `null` to pick last version automatically?

Reply:
I don't see why we can't, I'll test that out and adjust.

Reply:
Yeah, we don't have to pass it.